host-controller/src/main/java/org/jboss/as/host/controller/DomainModelControllerService.java

```diff
@@ -897,7 +897,8 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
                     Notification notification = new Notification(ModelDescriptionConstants.BOOT_COMPLETE_NOTIFICATION, PathAddress.pathAddress(PathElement.pathElement(CORE_SERVICE, MANAGEMENT),
                             PathElement.pathElement(SERVICE, MANAGEMENT_OPERATIONS)), ControllerLogger.MGMT_OP_LOGGER.bootComplete());
                     getNotificationSupport().emit(notification);
-                    bootstrapListener.printBootStatistics();
+                    bootstrapListener.printBootStatistics(true, environment.getDomainConfigurationFile().getMainFile().getName()
```

Comment:
environment.getDomainConfigurationFile() will be null if we are on an slave HC. You have to check this possibility.